package.json

```diff
@@ -2,19 +2,27 @@
   "name": "automata",
   "version": "1.0.0",
   "scripts": {
-    "dev": "tsc-watch --onSuccess \"node dist/index.js\"",
+    "dev": "pocketbase-typegen --env --out ./util/pbtypes.ts;tsc-watch --onSuccess \"node dist/index.js --dev\"",
     "build": "tsc"
   },
   "type": "module",
   "dependencies": {
+    "axios": "^1.5.0",
     "body-parser": "^1.20.2",
+    "cors": "^2.8.5",
     "dotenv": "^16.0.3",
     "execa": "^7.1.1",
-    "express": "^4.18.2"
+    "express": "^4.18.2",
+    "express-rate-limit": "^6.11.1",
+    "form-data": "^4.0.0",
```

Comment:
remove in favor of native URLSearchParams

Reply:
form-data you mean?